bgpd/bgp_nht.c

```diff
@@ -38,7 +38,8 @@ extern struct zclient *zclient;
 
 static void register_zebra_rnh(struct bgp_nexthop_cache *bnc);
 static void unregister_zebra_rnh(struct bgp_nexthop_cache *bnc);
-static int make_prefix(int afi, struct bgp_path_info *pi, struct prefix *p);
+static int make_prefix(int afi, safi_t safi, struct bgp_path_info *pi,
```

Comment:
why do we need to pass in the safi?  What is the goal here?  I don't see it ever being used.

Reply:
Fixed, thanks.